circle.yml

```diff
@@ -7,6 +7,9 @@ general:
 
 checkout:
   post:
+    - |
+        curl https://raw.githubusercontent.com/conda-forge/conda-forge-build-setup-feedstock/b2a07473a735ef6378be34c805a4b484b9d84579/recipe/ff_ci_pr_build.py | \
+            python - -v --ci "circle" "${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}" "${CIRCLE_BUILD_NUM}" "${CIRCLE_PR_NUMBER}"
```

Comment:
Also it might be worth adding this to `./scripts` as we did with `checkout_merge_commit.sh`. Would make the CircleCI log look a little nicer.

<img width="1552" alt="screen shot 2017-01-22 at 16 17 04" src="https://cloud.githubusercontent.com/assets/3019665/22186102/3e4aa162-e0be-11e6-8e8e-708a51100d65.png">